client/blocks/reader-post-card/index.jsx

```diff
@@ -177,43 +171,6 @@ class ReaderPostCard extends Component {
 		);
 		/* eslint-enable wpcalypso/jsx-classname-namespace */
 
-		// Set up post byline
-		let postByline;
-
-		if ( isDiscoverStream && discoverPost && discoverSite ) {
-			// create a post like object with some props from the discover post
-			const postForByline = Object.assign( {}, discoverPost || {}, {
-				date: post.date,
-				URL: post.URL,
-				primary_tag: post.primary_tag,
-			} );
-			postByline = (
-				<PostByline
-					post={ postForByline }
-					site={ discoverSite }
-					showSiteName={ true }
-					teams={ teams }
-					showFollow={ ! isDiscover }
-					showPrimaryFollowButton={ showPrimaryFollowButton }
-					followSource={ followSource }
-				/>
-			);
-		} else {
-			postByline = (
-				<PostByline
-					post={ post }
-					site={ site }
-					feed={ feed }
-					showSiteName={ showSiteName || isDiscover }
-					showAvatar={ ! compact }
-					teams={ teams }
-					showFollow={ ! isDiscover }
-					showPrimaryFollowButton={ showPrimaryFollowButton }
-					followSource={ followSource }
-				/>
-			);
-		}
```

Comment:
Is removing this `postByline` for the `else` case going to effect other pages or streams unexpectedly?

If we are looking to remove this for discover only, we could make this `null` in the conditional above, or just define it like this (the else condition) all the time and add a `! isDiscoverStream` to where we bring it in to render?

Reply:
> going to effect other pages or streams unexpectedly?

It looks like there is a regression on the following stream (posts in the feed are missing the top section that is probably this byline)

Reply:
I got a little carried away here! I've returned it and added condition below to skip including byline for discover stream posts.